packages/explat-react-client/.eslintrc.js

```diff
@@ -0,0 +1 @@
+module.exports = {};
```

Comment:
This shouldn't be necessary to add, right? I think `eslint` will just choose the configuration above it (correctly) in the directory structure.